frontend/src/app/shared/components/file-links/file-link-list-item/file-link-list-item.component.ts

```diff
@@ -101,6 +103,8 @@ export class FileLinkListItemComponent implements OnInit, AfterViewInit {
 
     this.fileLinkIcon = getIconForMimeType(this.originData.mimeType);
 
+    this.downloadAllowed = this.originData.mimeType !== 'application/x-op-directory';
```

Comment:
Out of curiosity, where is this `x-op-directory` type coming from? I assume it is a custom type used for all folders so that there are no other possible mimeTypes for folders depending on the OS?

Reply:
As there is no standard out there for a mimetype of directories (and yeah, basically every OS or web server does it differently), we introduced a custom mimetype, which is documented in our API specs.

Reply:
for future reference: in NC it is not depending on OS (anyway NC only runs on Linux) or WebServer but is coded into the NextCloud code https://github.com/nextcloud/server/blob/8af40542379df1338a07ec1628147bb79a92be4f/lib/public/Files/FileInfo.php#L67

Reply:
👍 Thanks for the clarification